code/game/objects/items/devices/PDA/PDA.dm

```diff
@@ -903,6 +903,29 @@ GLOBAL_LIST_EMPTY(PDAs)
 		playsound(src, 'sound/machines/pda_button1.ogg', 50, TRUE)
 	return TRUE
 
+/obj/item/pda/pre_attack(obj/target, mob/living/user, params)
+	if(!ismachinery(target))
+		return ..()
+	var/obj/machinery/target_machine = target
+	if(!target_machine.panel_open && !istype(target, /obj/machinery/computer))
```

Comment:
```suggestion
	var/obj/machinery/computer/target_machine = target
	if(!istype(target_machine) || !target_machine.panel_open)
```

Reply:
It looks like the original intent of the code is to succeed if either
- Target is a computer
- Target is a non-computer machine with the panel open

While this change makes it so the code succeeds only if **both**
- Target is a computer
- Target has panel open
Which I don't even know if is possible with computers?

Reply:
Ah I see.